deploying-an-existing-application-using-okteto-ui.md

```diff
@@ -0,0 +1,175 @@
+# Deploying An Existing Application Using Okteto UI
+
+In the previous blog post, you learned how to build a FastAPI CRUD application and deployed it to Okteto using the Okteto CLI tool. In this tutorial, you will be learning how to deploy the application directly from your Okteto dashboard.
```

Comment:
I'd like us to extend the introduction a bit, and talk about the value behind this. Points I Think we should mention:
- it's easy and fast (one click and you're done, instead of having to deal with CLIs, manifests, etc...)
- it helps you test end to end, the same your users will consume the application
- you can share with the rest of your team

Reply:
maybe add a gentle nudge to tell users to read post #1 first?

Reply:
Sure!

Reply:
Fixed :)